app/src/main/java/ai/elimu/analytics/db/RoomDb.java

```diff
@@ -13,24 +13,20 @@
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 
-import ai.elimu.analytics.dao.LetterAssessmentEventDao;
 import ai.elimu.analytics.dao.LetterSoundLearningEventDao;
 import ai.elimu.analytics.dao.StoryBookLearningEventDao;
 import ai.elimu.analytics.dao.WordAssessmentEventDao;
 import ai.elimu.analytics.dao.WordLearningEventDao;
-import ai.elimu.analytics.entity.LetterAssessmentEvent;
 import ai.elimu.analytics.entity.LetterSoundLearningEvent;
 import ai.elimu.analytics.entity.StoryBookLearningEvent;
 import ai.elimu.analytics.entity.VideoLearningEvent;
 import ai.elimu.analytics.entity.WordAssessmentEvent;
 import ai.elimu.analytics.entity.WordLearningEvent;
 import timber.log.Timber;
 
-@Database(version = 8, entities = {LetterAssessmentEvent.class, LetterSoundLearningEvent.class, WordLearningEvent.class, WordAssessmentEvent.class, StoryBookLearningEvent.class, VideoLearningEvent.class})
+@Database(version = 8, entities = {LetterSoundLearningEvent.class, WordLearningEvent.class, WordAssessmentEvent.class, StoryBookLearningEvent.class, VideoLearningEvent.class})
```

Comment:
@tuancoltech Please bump the database version from 8 to 9, and revert the changes in `app/schemas/ai.elimu.analytics.db.RoomDb/8.json`. Then, when you launch the app, a new DB schema export for version 9 of the database will be generated.
```suggestion
@Database(version = 9, entities = {LetterSoundLearningEvent.class, WordLearningEvent.class, WordAssessmentEvent.class, StoryBookLearningEvent.class, VideoLearningEvent.class})
```

Reply:
@jo-elimu Oh yes you're right. 
I've corrected that, and also add a migration strategy which drops the `LetterAssessmentEvent` when DB is upgraded.
Please review again.